src/data/event.py

```diff
@@ -115,6 +115,10 @@ def prize_currency(self) -> str:
             return plugin
         return SharlyChessConfig.default_prize_currency
 
+    @property
+    def override_unrated_rapide_blitz(self) -> bool:
+        return self.stored_event.override_unrated_rapide_blitz or False
```

Comment:
As the default value is True, shouldn't we return True for events migrated from previous releases?
```suggestion
        return self.stored_event.override_unrated_rapid_blitz if self.stored_event.override_unrated_rapid_blitz is not None else True
```

Reply:
I didn't want to change the behaviour of existing events.  My idea was to make it True for new events only.

Reply:
I understand.